### PR TITLE
Added_Image_Upload_feature

### PR DESCRIPTION
**Pull request by ManishShah120**
# Description [about the feature]:
- Added a Front-End feature to upload an image by any user, while creating a Blog-Post.
- Tweaked the UI of the detail page to display the image for any blog-post, if the user has uploaded any image while creating that blog post, else the blog post would be displayed as usual.

Fixes: #92 
 
# GIF [FEATURE]
ImageUploadFeature

### CODE REVIEW

**Comment by ManishShah120** [comment]
Hey @Kajol-Kumari, Can this be _merged_, else if there is anything I should change please let me know.

File: mysite/blog/templates/blog/home.html [hunk 1]
```diff
@@ -16,22 +16,22 @@
         <label class="slide-0">
             <h1 class="text-slide">{{ mostliked1.title }}</h1>
             <h5>{{mostliked1.content|truncatechars:70}}</h5>
-            <a href="{% url 'post-detail' mostliked1.id %}" class="readmorebutton">Read More</a>
+            <a href="{% url 'post-detail' slug=mostliked1.slug %}" class="readmorebutton">Read More</a>
```
Comment (Kajol-Kumari) [inline mysite/blog/templates/blog/home.html:19]: Hey @ManishShah120 I can see here the changes from `slug` PR. Can you please remove all these unnecessary changes?

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Good Work!
Functionality is working fine. Please incorporate the suggested changes and it'll be good to go.

**Comment by ManishShah120** [comment]
Hey @Kajol-Kumari , I have made the required changes and solved the conflict. Please have a look.

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :)

This can be merged now.

**Comment by ManishShah120** [comment]
@harsh-9in, I would love to see this **feature** added to this Project. Please let me know if there is any more changes I need to make for this _PR_ to get merged.

**Pull request merged**